conans/client/build/meson.py

```diff
@@ -43,7 +44,6 @@ def build_type(self, build_type):
                 'Set build type "%s" is different than the settings build_type "%s"'
                 % (build_type, settings_build_type))
         self._build_type = build_type
-        self.definitions.update(self._build_type_definition())
```

Comment:
`self._build_type_definition()` doesn't exists!

Reply:
I don't understand this diff, the code in `develop` has `self.definitions.update(build_type_definition(build_type, self.generator))` that indeed exist.

Reply:
In `cmake.py` it does, but not in `meson.py`

Reply:
LoL.
I think the `self.definitions` can be removed from meson too, it is not used nor documented.

Reply:
Yes, remove ``self.definitions`` if not used nor documented.